packages/palette/src/elements/Radio/Radio.tsx

```diff
@@ -139,7 +143,7 @@ const Container = styled(Flex)<{
         ${RADIO_STATES.focus}
 
         // Radio
-        > div:first-of-type {
+        > div:nth-of-type(2) {
```

Comment:
I realize this is more or less how this was but it's a tad more brittle. Can we target by nesting the targeted component? `${RadioDot} {`